index.html

```diff
@@ -303,6 +330,31 @@ <h2>Dictionary <code><a>RTCRtpEncodingParameters</a></code> Members</h2>
       </dd>
     </dl>
   </section>
+  <section id="rtcrtpcodeccapability-dictionary">
+    <h3>
+      <dfn>RTCRtpCodecCapability</dfn> extensions
+    </h3>
+    <p>
+      The <code><a href="https://w3c.github.io/webrtc-pc/#rtcrtpcodeccapability">RTCRtpCodecCapability</a></code>
+      dictionary is defined in [[WEBRTC]]. This document extends that dictionary
+      by adding an additional boolean flag.
+    </p>
+    <pre class="idl">partial dictionary RTCRtpCodecCapability {
+      boolean hardwareAccelerated;
```

Comment:
I still think returning a boolean for "hardware accelerated" is a Bad Idea.
It doesn't tell you if using that hardware is going to be more useful than not using it; it just makes you think that it is so.